cmd/sts-errors.go

```diff
@@ -102,6 +103,11 @@ var stsErrCodes = stsErrorCodeMap{
 		Description:    "The client grants that was passed is expired or is not valid. Get a new client grants token from the identity provider and then retry the request.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrSTSKerberosExpiredToken: {
+		Code:           "ExpiredToken",
+		Description:    "The Kerberos service ticket had an expired token",
```

Comment:
Should it be not `has` instead of `had` - since it's happening in present sense ?